README.md

```diff
@@ -1 +1,285 @@
-# terraform-google-bigquery-dataset-iam
\ No newline at end of file
+[<img src="https://raw.githubusercontent.com/mineiros-io/brand/3bffd30e8bdbbde32c143e2650b2faa55f1df3ea/mineiros-primary-logo.svg" width="400"/>][homepage]
+
+[![Terraform Version][badge-terraform]][releases-terraform]
+[![Google Provider Version][badge-tf-gcp]][releases-google-provider]
+[![Join Slack][badge-slack]][slack]
+
+# terraform-terraform-google-bigquery-dataset-iam
+
+A [Terraform] module for [Google Cloud Platform (GCP)][gcp].
+
+**_This module supports Terraform version 1
+and is compatible with the Terraform Google Provider version 3._**
+
+This module is part of our Infrastructure as Code (IaC) framework
+that enables our users and customers to easily deploy and manage reusable,
+secure, and production-grade cloud infrastructure.
+
+- [Module Features](#module-features)
+- [Getting Started](#getting-started)
+- [Module Argument Reference](#module-argument-reference)
+  - [Top-level Arguments](#top-level-arguments)
+    - [Module Configuration](#module-configuration)
+    - [Main Resource Configuration](#main-resource-configuration)
+    - [Extended Resource Configuration](#extended-resource-configuration)
+- [Module Attributes Reference](#module-attributes-reference)
+- [External Documentation](#external-documentation)
+- [Module Versioning](#module-versioning)
+  - [Backwards compatibility in `0.0.z` and `0.y.z` version](#backwards-compatibility-in-00z-and-0yz-version)
+- [About Mineiros](#about-mineiros)
+- [Reporting Issues](#reporting-issues)
+- [Contributing](#contributing)
+- [Makefile Targets](#makefile-targets)
+- [License](#license)
+
+## Module Features
+
+In contrast to the plain `terraform_google_bigquery_dataset_iam_binding`, `terraform_google_bigquery_dataset_iam_member` resources this module has better features.
+While all security features can be disabled as needed best practices
+are pre-configured.
```

Comment:
@Hanymin please reword this.. what does "better features mean" what security features? what best practices? this all doe snot apply to this module..

please describe what features the module supports in more detail. like authoratitive mode vs non-authoritative